backend/models.py

```diff
@@ -12,19 +12,13 @@ class User(db.Model):
     name = db.Column(db.String(500))
     avatar_url = db.Column(db.String(500), nullable=True)
     faculty = db.Column(db.Boolean)
-    anonymous_by_default = db.Column(db.Boolean)
-    autosave_timeout = db.Column(db.Integer)
-    autosave_enabled = db.Column(db.Boolean)
     random_seed = db.Column(db.LargeBinary(32))
 
     def __init__(self, id, name, **kwargs):
         self.id = id
         self.name = name
         self.avatar_url = kwargs.get("avatar_url", None)
         self.faculty = kwargs.get("faculty", None)
-        self.anonymous_by_default = kwargs.get("anonymous_by_default", False)
```

Comment:
It looks like `anonymous_by_default` is still used: https://github.com/mjec/rc-niceties/blob/4caabafb46f38aafb4f558a17428b7bdda2365e2/backend/api.py#L325

So I think we need to remove that use as part of this change!

Reply:
I must have deleted my comments, but yes you're right, I added this in.